tests/strategies/test_parse_image.py

```diff
@@ -0,0 +1,74 @@
+"""Test the image formats in the image parse strategy."""
+from pathlib import Path
+
+import numpy as np
+from PIL import Image
+import pytest
+
+
+@pytest.mark.parametrize("crop_rect", [None, (100, 100, 250, 200)])
+@pytest.mark.parametrize(
+    "test_file, target_file",
+    (
```

Comment:
Comment:
No actual need to test a lot of file formats here. Handling of different file formats should be unit tested in PIL (and is even tested in oteapi-core). Here I think it should be sufficient to check that we can load a single image and that we can correctly transfer cropping parameters to the image strategy in oreapi-core.

But this rigorous tests do not harm either (except for slowing the tests down), so let us keep them for now.